### DISCUSSION TITLE
랜덤한 id 생성하기

### DISCUSSION THREAD

**Discussion by jkea1** [prgrms-fe-devcourse/FEDC4_HONKOK_JunilHwang#47] [General]
id 생성을 라이브러리 사용을 안하고 커스텀 함수로 만든다고 들었던거 같은데 내일 같이 이야기 해봅시다.

**Comment by backward99** [comment]
o0o? 그런 것도 있군요..!

**Comment by jgjgill** [comment]
gpt에 물어봤습니다..!
https://chat.openai.com/share/6be46ac0-361d-4151-ad16-c0a36fbaac1d

uuid 라이브러리를 사용해도 되는데 저는 안깔아도 되지 않나 싶어서 이야기했었네요..!
uuid 라이브러리를 사용해도 상관없기에 공유만 하면 될 것 같아요!
